### DISCUSSION TITLE
Enhancement Request: "New Mail" used for undismissed notifications

### DISCUSSION THREAD

**Discussion by HeofHIShirts** [InfiniTimeOrg/InfiniTime#1616] [Ideas]
I'm describing the behavior as it is referred in the Terminal watchface, but I'm sure there are similar indicators in the other watch faces.

Current behavior: When notifications are set to silent on the watch, the presence of new notifications is indicated by the phrase "You have new mail" on the terminal watchface. Swiping up to read the notifications dismisses this message.

When silent is not set, new notifications appear on the screen briefly and then disappear, and the "new message" message does not show up.

Desired change: Now that InfiniTime, with 1.10, offers the ability to dismiss notifications with a gesture, I would like the "new notifications" message to appear when the number of notifications available to read is greater than 0/0, regardless of whether the watch is in silent or unsilent mode, and to only disappear after all the available notifications have been dismissed.

Thank you for the consideration of this entrancement.

**Comment by Riksu9000** [comment]
We've decided to only consider this if the notifications are synced with a phone.
